src/components/header/nav.tsx

```diff
@@ -0,0 +1,64 @@
+import { Box, Link } from "@mui/material";
+import { Auth } from "../../atoms/auth";
+
+interface HomeNavProps { auth: Auth | undefined}
+
+const HomeNav = (props: HomeNavProps) => {
+    const { auth } = props;
+
+    interface NavButtonProps {
+        text: string, 
+        selected: boolean, 
+        sx_props: object
+    } 
+    const NavButton = (props: NavButtonProps) => {
+      const {text, selected, sx_props} = props;
+      return (
+        <Link 
+            href="#"
+            underline="none"
+            sx={ Object.assign({
+          "background-color": (selected ? "#dee2e5" : ""),
+          "height": "100%",
+          "padding": "15px"
+        }, sx_props)}>
+          { text }
+        </Link>
+      );
+    }
+
+    return (
+        <Box sx={{
+            width:"100%",
+            display:"flex",
+            justifyContent:"space-between",
+            "padding-top": "16px",
+            "padding-right": "22px",
+            "padding-bottom": "14px",
+            "padding-left": "0px"
+          }}>
+            <Box>
+              <NavButton 
+                  text="Home" 
+                  selected={ true } 
+                  sx_props={{
+                "border-top-left-radius": "15px",
+                "border-bottom-left-radius": "15px"
+              }}/>
+              <NavButton text="Admin" selected={false} sx_props={{}}/>
```

Comment:
This makes me think sx_props should be typed as optional in the interface you create